index.d.ts

```diff
@@ -1,2 +1,2 @@
-declare function clipboardCopy (text: string): boolean
+declare function clipboardCopy (text: string): Promise
```

Comment:
The return value should be specified as `Promise<void>`

Reply:
Cool, I was guessing and hoping you'd correct me :)